### PR TITLE
Shopfront: On non-shop tab, when changing OC, switch to shop tab and hides non-shop tab content

### LINKED ISSUE #11223: Notices tab on shopfront persisting after selecting an order cycle
## Description

Opening a shopfront shows the Notices screen. Customers can click the Shop tab to start ordering where the shop has a single open order cycle. For shops with multiple order cycles, the customer must first select which order cycle they want to order from. Once they've selected an order cycle, the site used to redirect to the Shop tab. Now, whilst the tab selection changes, the Notices information persists above the shopfront. Clicking the Shop tab doesn't remove the Notices information and it appears to users that the shop cannot be accessed unless they scroll all the way down past lengthy text. Unsure when exactly this was introduced, but within the last couple of weeks.

## Expected Behavior

Selecting an order cycle should redirect users to the Shop tab, without any Notices information being displayed.

## Actual Behaviour

Selecting an order cycle does not remove the Notices information

## Steps to Reproduce

1. Go to a shop with multiple open order cycles e.g. [this one](https://openfoodnetwork.org.uk/sole-of-discretion/shop#/)
2. Observe link defaults to Notices tab
3. Select an order cycle
4. Observe tab highlight changes to Shop tab, but Notices text does not disappear. Shop tab content appears beneath Notices content. Can be forced to fix by clicking Notices again, then back to Shop

## Animated Gif/Screenshot

Notice tab highlight changing, but Notices not disappearing on Shops tab.
Notices not minimising

## Workaround

User must scroll all the way down past notices text or click Notices tab, then click back to Shop tab. This is unintuitive and frustrating for users

## Severity
bug-s4: it's annoying, but you can use it

## Your Environment

* Version used:
* Browser name and version:
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

- Closes #11223 

The title of the PR is pretty cryptic, but it's well explained in the linked issue, and you can see the result here:

#### What should we test?
- Go to a shop with multiple open order cycles
- Switch to a different tab (could be the white labelled one)
- Select an order cycle
- Check that we are _redirected_ to shop tab/panel
- Check that content from the previous tab is hidden

#### Release notes

Changelog Category: User facing changes

### CODE REVIEW

**Review by mkllnk — APPROVED** [review]

**Review by dacook — APPROVED** [review]
😁 Simple fix!

No need for second review.

**Pull request merged**
